pkgs/samples/default.nix

```diff
@@ -229,6 +229,7 @@ let
 
     unpackCmd = "dpkg -x $src source";
     sourceRoot = "source/usr/src/jetson_multimedia_api";
+    separateDebugInfo = true;
```

Comment:
This isn't a valid option for upstream nixpkgs.

Reply:
I think it is now: https://github.com/nixos/nixpkgs/blob/9306e3cc87e0bcf686ededc6060add6d3245cca2/doc/stdenv/stdenv.chapter.md?plain=1#L1088